manifests/bootkube/bootstrap-manifests/bootstrap-apiserver.yaml

```diff
@@ -4,10 +4,10 @@ metadata:
   name: bootstrap-kube-apiserver
   namespace: {{ .Namespace }}
   labels:
-    openshift.io/control-plane: true
-    openshift.io/component: api
+    openshift.io/control-plane: "true"
```

Comment:
why we need quotes?

Reply:
because YAML 🤢 @deads2k's merging algorithm uses type-less yaml/json decoding which makes a boolean out of `true`.